src/Microsoft.AspNetCore.Dispatcher/Constraints/AlphaDispatcherValueConstraint.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Dispatcher
+{
+    /// <summary>
+    /// Constrains a dispatcher value parameter to contain only lowercase or uppercase letters A through Z in the English alphabet.
+    /// </summary>
+    public class AlphaDispatcherValueConstraint : RegexDispatcherValueConstraint
```

Comment:
ported because it's used in tests? Seems fine for now. We will eventually redesign some of the constraints, but this is a good thing to do if it unblocks you

Reply:
Yeah, I only ported the ones that are used in tests.